### ISSUE TITLE
Markdown isn't showing/rendering after cleaning the page

### ISSUE THREAD

**Issue by mkhachfeh** [flet-dev/flet#407]
**Description**

My app contains multiple logical pages (I am cleaning the page and adding the needed objects for the needed page).

I am using Markdown to render some texts within one of these pages (help page).
I saw that when I run the page containing the markdown object, it renders correctly, but if I go to another page within the same app, and go back to the page containing the Markdown, it shows nothing.

**Code example to reproduce the issue:**
```
    def help_page(e):
        page.clean()
        add_basic_buttons(page) ### Func to add the basic buttons above
        help_msg = """ 
## About...

- aa
- bb
- cc
        """
        page. Add(
            Markdown(
                value=help_msg,
                selectable=True,
                extension_set="commonMark",
                on_tap_link=lambda e: page.launch_url(e.data),
                expand=True,
                )
        )
```

**Describe the results you received:**
As said, first click on 'Help page' it shows the below rendered:
<img width="796" alt="Screenshot 2022-09-28 at 14 59 14" src="https://user-images.githubusercontent.com/40859905/192784903-6571baa1-de8f-4e54-bba1-1343fd165949.png">

If I go to another page, and then come back to 'Help page', it doesn't show the Markdown

**Describe the results you expected:**

I expect every time to render the Markdown and show the result.

**Flet version (`pip show flet`)**:
Name: flet
Version: 0.1.52
Summary: Flet for Python - easily build interactive multi-platform apps in Python
Home-page: 
Author: 
Author-email: Appveyor Systems Inc. <hello@flet.dev>
License: MIT
Requires: beartype, repath, websocket-client
Required-by: 

**Operating system**:

macOS

**Additional environment details:**
NA

**Comment by FeodorFitsner** [comment]
Thanks for starting an issue with providing the details!

1) Could you please upgrade to the latest Flet 0.1.60?
2) *if I go to another page within the same app, and go back to the page containing the Markdown* - provided code is not enough to see how do you navigate between pages. Could you please provide more complete repro?

**Comment by mkhachfeh** [comment]
hmm but now tested again for your example mentioned here:
https://flet.dev/docs/controls/markdown

And seems the scrolling isn't working anymore

**Comment by mkhachfeh** [comment]
omg, so scrolling has been resolved on flet==0.1.60 by removing `page.scroll` from the code
